android/src/main/java/com/maskedtextinput/views/AdvancedTextInputMaskDecoratorView.kt

```diff
@@ -74,6 +75,12 @@ class AdvancedTextInputMaskDecoratorView(
             affinityCalculationStrategy = affinityCalculationStrategy,
             allowedKeys = allowedKeys,
           )
+
+        val defaultValue = this.defaultValue
+
+        if (!defaultValue.isNullOrEmpty()) {
+          maskedTextChangeListener?.setText(defaultValue, true)
+        }
```

Comment:
defaultValue?.let { maskedTextChangeListener?.setText(it, true) }?

Maybe move it to reusable function if we decide to keep prev style?